source/common-tasks/general.markdown

```diff
@@ -9,11 +9,17 @@ This section provides tasks that do not depend on a specific Home Assistant inst
 
 ## Defining a custom polling interval
 
-Creating an automation for polling gives you more flexibility on when to poll.
+If you want to define a specific interval at which your device is being polled for data, you can disable the default polling interval and create your own polling automation.
 
-Why use an automation instead of changing the integration's configuration for polling?
+## What is data polling?
```

Comment:
Side note: It would be cool to link to this from a term so we can easily reuse this explanation in the integration pages.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/35366